exercises/concept/freelancer-rates/.docs/instructions.md

```diff
@@ -37,5 +37,5 @@ The discount is always passed as a number, where `42%` becomes `0.42`. The resul
 
 ```javascript
 priceWithMonthlyDiscount(89, 230, 0.42);
-// => 97972
+// => 97971
```

Comment:
`97972` is the correct value.

Reply:
I tried with `Math.round()` and `.toFixed(0)`
Always getting `97971`

```
const dayRate = (hourlyRate) => {
  return hourlyRate * 8;
};

const priceWithMonthlyDiscount = (hourlyRate, days, discount) => {
  let fullMonths = Math.round(days / 22);
  let costPerMonthWithDiscount = 22 * dayRate(hourlyRate) * (1 - discount);
  let remainingDays = days - fullMonths * 22;
  let costForRemainingDays = remainingDays * dayRate(hourlyRate);
  return Math.round(
    costPerMonthWithDiscount * fullMonths + costForRemainingDays
  );
};

console.log(priceWithMonthlyDiscount(89, 230, 0.42));
```

Reply:
That's because there is a tiny mistake in the code provided. 

From the instructions:

> In these cases, the freelancer decides to offer a discount for every full month [...]

Can you spot what's different in this sentence from the code provided by you?